pkg/apis/compliance/v1alpha1/tailoredprofile_types.go

```diff
@@ -6,6 +6,18 @@ import (
 
 // FIXME: move name/rationale to a common struct with an interface?
 
+// OutdatedReferenceValidationDisable a label is used to disable validation of outdated references
+const OutdatedReferenceValidationDisable = "compliance.openshift.io/outdated-reference-validation-disable"
```

Comment:
nit: for readability I would consider putting the verb first (`DisableOutdatedReferenceValidation`)